README.md

```diff
@@ -68,6 +71,15 @@ In order to run this project you need:
 - Git scm.
 
 ### Setup
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://jazzy-crostata-9099d2.netlify.app/)
+
+
+## 🎥 Video Link <a name="live-demo"></a>
+-[Video describing the project](https://www.loom.com/share/b475c651da16461296bbd861d35e2b68)
 
 Clone this repository to your desired folder:
 
```

Comment:
- [ ] The License link and issues page link is broken in the README file. Please make sure to fix this. It will make the README file more professional.